recipes/google-cloud-cpp/all/conanfile.py

```diff
@@ -70,13 +70,13 @@ def source(self):
         tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
 
     def requirements(self):
-        self.requires('protobuf/3.20.0')
-        self.requires('grpc/1.45.2')
+        self.requires('protobuf/3.21.4')
```

Comment:
Give the fact the old generated code I am curious if this would be a next step 🤔 

```suggestion
        self.requires(if tools.Version(self.version) > "1.40.0" 'protobuf/3.21.4' else 'protobuf/3.20.0')
```